includes/class-wc-google-analytics.php

```diff
@@ -475,35 +475,39 @@ public function add_to_cart() {
+		if ( 'yes' === $this->ga_gtag_enabled ) {
+			$this->get_tracking_instance()->add_to_cart( $product );
+		} else {
+			// Add single quotes to allow jQuery to be substituted into _trackEvent parameters
+			$parameters             = array();
+			$parameters['category'] = "'" . __( 'Products', 'woocommerce-google-analytics-integration' ) . "'";
+			$parameters['action']   = "'" . __( 'Add to Cart', 'woocommerce-google-analytics-integration' ) . "'";
+			$parameters['label']    = "'" . esc_js( $product->get_sku() ? __( 'ID:', 'woocommerce-google-analytics-integration' ) . ' ' . $product->get_sku() : '#' . $product->get_id() ) . "'";
 
-		if ( ! $this->disable_tracking( $this->ga_enhanced_ecommerce_tracking_enabled ) ) {
+			if ( ! $this->disable_tracking( $this->ga_enhanced_ecommerce_tracking_enabled ) ) {
 
-			$item = "{";
+				$item = '{';
 
-			if ( $product->is_type( 'variable' ) ) {
-				$item .= "'id': google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ] !== undefined ? google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ].id : false,";
-				$item .= "'variant': google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ] !== undefined ? google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ].variant : false,";
-			} else {
-				$item .= "'id': '" . $this->get_tracking_instance()->get_product_identifier( $product ) . "',";
-			}
+				if ( $product->is_type( 'variable' ) ) {
+					$item .= "'id': google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ] !== undefined ? google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ].id : false,";
+					$item .= "'variant': google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ] !== undefined ? google_analytics_integration_product_data[ $('input[name=\"variation_id\"]').val() ].variant : false,";
+				} else {
+					$item .= "'id': '" . $this->get_tracking_instance()->get_product_identifier( $product ) . "',";
+				}
 
-			$item .= "'name': '" . esc_js( $product->get_title() ) . "',";
-			$item .= "'category': " . $this->get_tracking_instance()->product_get_category_line( $product );
-			$item .= "'quantity': $( 'input.qty' ).val() ? $( 'input.qty' ).val() : '1'";
-			$item .= "}";
+				$item .= "'name': '" . esc_js( $product->get_title() ) . "',";
+				$item .= "'category': " . $this->get_tracking_instance()->product_get_category_line( $product );
+				$item .= "'quantity': $( 'input.qty' ).val() ? $( 'input.qty' ).val() : '1'";
+				$item .= '}';
 
-			$parameters['item'] = $item;
+				$parameters['item'] = $item;
 
-			$code = "" . $this->get_tracking_instance()->tracker_var() . "( 'ec:addProduct', " . $item . " );";
-			$parameters['enhanced'] = $code;
-		}
+				$code                   = $this->get_tracking_instance()->tracker_var() . "( 'ec:addProduct', " . $item . ' );';
```

Comment:
Since we are already using double quotes and a single variable we can just interpolate instead of concat. In my opinion slightly easier to read.

 ```suggestion
				$code                   = $this->get_tracking_instance()->tracker_var() . "( 'ec:addProduct', {$item} );";
```

Reply:
I've [updated](https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/240/commits/859677a7cc31b488bdb249c041a9cfe03c2a969f)* that line of code with your suggestion.